scripts/zones/East_Ronfaure_[S]/mobs/Battrap.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------
+-- Area: East Ronfaure [S]
+--  MOB: Battrap
+-- Note:  Place Holder for Goblintrap NM
+-- !pos 156 0 -438 81
+-----------------------------------
+
+require("scripts/zones/East_Ronfaure_[S]/MobIDs");
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+end;
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
+
+    phOnDespawn(mob,GOBLINTRAP_PH,5,310);
```

Comment:
onMobDeath should be an empty function (it must exist, though, or the core complains)
```
function onMobDeath(mob, player, isKiller)
end;
```

phOnDespawn() should be called in onMobDespawn().  According to wiki the cooldown is 60 minutes, or 3600 seconds.

```
function onMobDespawn(mob)
    phOnDespawn(mob,GOBLINTRAP_PH,5,3600); -- 1 hour
end;
```